projects/components/src/utils/test/datagrid/datagrid.wo.ts

```diff
@@ -168,17 +209,24 @@ export class ClrDatagridWidgetObject extends WidgetObject<ClrDatagrid> {
     }
 
     /**
-     * Presses the button at the given {@param index} on the top of the grid.
+     * Gives the text of the button with the given {@param buttonClass}
+     */
+    getButtonText(buttonClass: string): string {
+        return this.getText(`button.${buttonClass}`);
+    }
+
+    /**
+     * Says if the button with the given {@param buttonClass} is disabled.
      */
-    pressTopButton(index: number): void {
-        this.click(`clr-dg-action-bar button:nth-of-type(${index + 1})`);
+    getButtonDisabled(buttonClass: string): string {
```

Comment:
Usually, getters that return booleans are name `is` or `has` or a verb that you typically reply with true or false

`isButtonDisabled()`

Consider inverting the logic to avoid things double negatives like `!isButtonDisabled()`